ui/packages/shared/components/src/ParcaContext/index.tsx

```diff
@@ -19,7 +19,7 @@ interface Props {
   perf: {
```

Comment:
Should we make this a `perf?:` instead of the `| null` at the end? 
I believe, the `| null`, would still enforce a `perf: null` in the passed object.

Reply:
TIL: you can do this `perf?`. that sgtm